loadgen/cmd/otelbench/main.go

```diff
@@ -98,6 +98,7 @@ func runBench(signal, exporter string, concurrency int) testing.BenchmarkResult
 
 		err := RunCollector(context.Background(), stop, configs(exporter, signal, b.N, concurrency), logsDone, metricsDone, tracesDone)
 		if err != nil {
+			fmt.Println(err.Error())
```

Comment:
```suggestion
			fmt.Fprintln(os.Stderr, err.Error())
```
use Fprintln to Stderr, as Stdout is for benchmark output